src/Argument.h

```diff
@@ -56,13 +56,11 @@ struct Argument {
      * By default, they are left unset, implying "no default, no min, no max". */
     Expr def, min, max;
 
-  int stride_multiples[4];
     Argument() : kind(InputScalar), dimensions(0) {}
     Argument(const std::string &_name, Kind _kind, const Type &_type, uint8_t _dimensions,
                 Expr _def = Expr(),
                 Expr _min = Expr(),
-                Expr _max = Expr(),
-                int *_stride_multiples = nullptr) :
```

Comment:
These changes should not be happening, this file should be unchanged from hexagon.